api/pom.xml

```diff
@@ -79,8 +79,12 @@
       <artifactId>logback-classic</artifactId>
       <scope>test</scope>
     </dependency>
-    
 
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
```

Comment:
We don't want to depend on Jackson in the API module. Instead we need to create a DTO for the SearchResults object, similar to what we do in https://github.com/OpenNMS/newts/blob/master/rest/src/main/java/org/opennms/newts/rest/ResultDescriptorDTO.java.